src/monitorCloudFormationStack.test.ts

```diff
@@ -0,0 +1,15 @@
+// import { APIGatewayEvent, Handler, Callback, Context } from "aws-lambda";
```

Comment:
/nit can we just remove this line if it's not being used?

Reply:
done